No member named 'default_arg_values' in 'GDScriptParser::FunctionNode'

### Godot version

ffd64505c51cb9734ad68f52277c807a9b759338

### System information

Fedora 35, Nvidia 3000 series

### Issue description

Can't build web platform.

```
modules/gdscript/gdscript_analyzer.cpp:1172:33: error: no member named 'default_arg_values' in 'GDScriptParser::FunctionNode'
                        valid = valid && p_function->default_arg_values.size() >= default_par_count + par_count_diff;
```



### Steps to reproduce

```
scons werror=no platform=javascript target=release_debug -j`nproc` use_lto=no deprecated=no use_llvm=yes builtin_freetype=yes initial_memory=256 CCFLAGS='-fno-stack-protector -fno-exceptions' custom_modules=../godot-modules-groups
```

### Minimal reproduction project

Use the above scons command.

--- a/modules/gdscript/gdscript_analyzer.cpp
+++ b/modules/gdscript/gdscript_analyzer.cpp
@@ -1154,7 +1154,7 @@ void GDScriptAnalyzer::resolve_function_signature(GDScriptParser::FunctionNode *
 		GDScriptParser::DataType return_type = resolve_datatype(p_function->return_type);
 		p_function->set_datatype(return_type);
 
-#ifdef DEBUG_ENABLED
+#ifdef TOOLS_ENABLED
 		// Check if the function signature matches the parent. If not it's an error since it breaks polymorphism.
 		// Not for the constructor which can vary in signature.
 		GDScriptParser::DataType base_type = parser->current_class->base_type;

